package.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "stylelint-create",
+  "version": "0.0.1",
+  "description": "Initialization wizard for Stylelint, a linter that helps you avoid errors and enforce conventions in your styles.",
+  "keywords": [
+    "css-in-js",
+    "css",
+    "less",
+    "lint",
+    "linter",
+    "markdown",
+    "sass",
+    "scss",
+    "stylelint",
+    "sugarss"
+  ],
+  "homepage": "https://github.com/stylelint/stylelint-create#readme",
+  "bugs": {
+    "url": "https://github.com/stylelint/stylelint-create/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stylelint/stylelint-create.git"
+  },
```

```suggestion
  "repository": "stylelint/create-stylelint",
```

Fields are inferred.